src/Opdex.Platform.WebApi/Models/Responses/LiquidityPools/LiquidityPoolResponseModel.cs

```diff
@@ -26,7 +26,7 @@ public class LiquidityPoolResponseModel
     /// Transaction fee percentage for swaps.
     /// </summary>
     /// <example>0.3</example>
-    public decimal TransactionFee { get; set; }
```

Comment:
Renamed `TransactionFee` to `TransactionFeePercent` in the liquidity pool response